biz.aQute.bnd.gradle/bnd.bnd

```diff
@@ -22,6 +22,9 @@ Bundle-Description: The Bnd Gradle plugin.
 	biz.aQute.resolve;version=latest, \
 	biz.aQute.repository;version=latest
 
+-testpath: \
```

Comment:
The Junit 5 commit must not be part of this PR.

Reply:
The JUnit 5 PR has been merged now. So this commit should be removed from this PR.

Reply:
Yes. The plan was to rebase onto matter once the JUnit Platform change was merged.